features/step_definitions/api/metrics_steps.rb

```diff
@@ -59,6 +59,18 @@ def map_or_unmap_metric(metric, mapped)
   assert_equal '', find_mapped_cell_in_table(name).text
 end
 
+Then /^I should (not )?see "([^"]*)" on the (metrics|methods) tab$/ do |negate, metric_name, tab_name|
```

Comment:
Please avoid using _I this_ and _I that_ when writing new scenarios/steps.

I think this step is unnecessary, we can use `I see the following metrics:` or simply `I should see "Pasta" within the metrics table`.